File: Core/Entities/Product.cs
namespace Core.Entities
{
    public class Product : BaseEntity
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public int Price { get; set; }
        public string PictureUrl { get; set; }
        public ProductType ProductType { get; set; }
        public int ProductTypeId { get; set; }
        public ProductBrand ProductBrand { get; set; }
        public int ProductBrandId { get; set; }
        public string Specs { get; set; }
        public bool? Used { get; set; }
        public string LocalId { get; set; }
    }
}

File: client/src/app/shared/models/product.ts
import { Photo } from "./photo";

export interface product {
    id: number;
    name: string;
    description: string;
    price: number;
    photoUrl: string;
    productType: string;
    productBrand: string;
    localId: string;
    specs: string;
    photos: Photo[];

}

